setup.py

```diff
@@ -493,6 +494,9 @@ def pxd(name):
                     'pxdfiles': ['_libs/src/util', '_libs/lib'],
                     'depends': tseries_depends,
                     'sources': npdt_srces},
+    '_libs.tslibs.np_datetime': {'pyxfile': '_libs/tslibs/np_datetime',
+                               'depends': tseries_depends[:2],
+                               'sources': npdt_srces},
```

Comment:
just spell things out, ``np_datetime_sources`` is much more readable (and change all references)